MetasysServices/NetworkDevices/NetworkDeviceServiceProvider.cs

```diff
@@ -58,7 +58,17 @@ public async Task<IEnumerable<MetasysObject>> GetAsync(string type = null)
             var response = await this.GetAllAvailablePagesAsync("networkDevices", new Dictionary<string, string> { { "type", type } }).ConfigureAwait(false);
             return ToMetasysObject(response, Version, MetasysObjectTypeEnum.Object);
         }
-
+        /// <inheritdoc/>
+        public IEnumerable<MetasysObject> GetByClassification(string classification = null)
+        {
+            return GetByClassificationAsync(classification).GetAwaiter().GetResult();
+        }
+        /// <inheritdoc/>
+        public async Task<IEnumerable<MetasysObject>> GetByClassificationAsync(string classification = null)
```

Comment:
Would it be possible to define this method as overload of  the method 'Get()' (adding the classification parameter) ?
Also, what about define a specific 'enum' (e.g. 'NetworkDeviceClassificationEnum') to better specify the parameter 'classification' ?

Reply:
Added method GetAsync to INetworkDeviceService and NetworkDeviceServiceProvider.cs and created enum 'NetworkDeviceClassificationEnum'